assets/queries/openAPI/path_parameter_not_required/query.rego

```diff
@@ -0,0 +1,39 @@
+package Cx
+
+import data.generic.openapi as openAPILib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openAPILib.checkOpenAPI(doc) != "undefined"
+	paths := doc.paths[name]
+	param := paths.parameters[n]
+	param.in == "path"
+	object.get(param, "required", "undefined") == "undefined"
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("openapi.paths.{{%s}}.parameters.name={{%s}}", [name, param.name]),
```

Comment:
Since it is a missing attribute from parameters, shouldn't be more clear to point to parameters instead name?

Reply:
Since parameters is an array we can point to the element of the array missing the 'required' field using the element name

Reply:
Does make sense